### PR TITLE
Checkup, setup: Add setup context

### PR DESCRIPTION
**Pull request by orelmisan**
Currently, the setup is using the global context, which has the timeout set by the user.

In case the checkup had created in VMIs, there is a wait for both VMIs to be ready, that does not fail until the global timeout.

If the user-supplied timeout is long, the checkup will wait for a long time before failing.

Add a context with a ten minutes timeout for the setup phase, derived from the global context.

Fixes [BZ 2243672](https://bugzilla.redhat.com/show_bug.cgi?id=2243672).

### CODE REVIEW

**Comment by orelmisan** [comment]
Tested against an OpenShift Virtualization 4.14 cluster.

File: pkg/internal/checkup/checkup.go [hunk 1]
```diff
@@ -86,14 +86,18 @@ func New(client kubeVirtVMIClient, namespace string, checkupConfig config.Config
 }
 
 func (c *Checkup) Setup(ctx context.Context) (setupErr error) {
+	const setupTimeout = 10 * time.Minute
```
Comment (RamLavi) [inline pkg/internal/checkup/checkup.go:89]: Are you sure 10 minutes are OK even if the images are not yet pulled?

Comment (orelmisan) [reply pkg/internal/checkup/checkup.go:89]: We can increase it to 15 mins.
WDYT?

Comment (RamLavi) [reply pkg/internal/checkup/checkup.go:89]: I'm not sure really, we need to measure it and give some extra to be sure. 
It's possible that 10 minutes is enough

Comment (orelmisan) [reply pkg/internal/checkup/checkup.go:89]: It worked for me in the 10 minutes time frame (including image pulling).

Comment (RamLavi) [reply pkg/internal/checkup/checkup.go:89]: OK then that's enough for now. 10 minutes should be more than enough.

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**